machine.c

```diff
@@ -1091,6 +1114,10 @@ SDL_bool emu_event( SDL_Event *ev, struct machine *oric, SDL_bool *needrender )
           via_init( &oric->via, oric, VIA_MAIN );
           via_init( &oric->tele_via, oric, VIA_TELESTRAT );
           acia_init( &oric->tele_acia, oric );
+#ifdef WIN32
+          oric->tele_ch376=ch376_oric_init();
+          ch376_oric_config(oric->tele_ch376);
```

Comment:
Maybe add a test to check the value of oric->tele_ch376 against NULL in case of memory allocation or context initialization failure in function ch376_create() called by ch376_oric_init()